BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -295,4 +297,17 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       rewardsSummaryView.alpha = 1.0
     }
   }
+  
+  private func tappedDisclaimerLink(_ url: URL) {
+    switch url.path {
+    case "/terms":
+      state.delegate?.loadNewTabWithURL(URL(string: "https://brave.com/terms-of-use/")!) //swiftlint:disable:this force_unwrapping
```

Comment:
Unwrap, and extract the url string somewhere? We use it in two places, would be good to put it somewhere so when it changes it will require only one place to fix